src/libraries/System.DirectoryServices/src/System/DirectoryServices/ActiveDirectory/ActiveDirectorySite.cs

```diff
@@ -7,6 +7,8 @@
 using System.Text;
 using System.Diagnostics.CodeAnalysis;
 
+using Kernel32 = Interop.Kernel32;
```

Comment:
We generally don't do this elsewhere.  Can we just spell out Interop.Kernel32 at each use site?

Reply:
I did it because there is also `System.DirectoryServices.Interop` and having `global::Interop.*` everywhere looked obnoxious to me. I can do `global::Interop.*` at each use site if that is preferred.